mars/tensor/expressions/random/rand.py

```diff
@@ -68,6 +68,8 @@ def rand(random_state, *dn, **kw):
            [ 0.37601032,  0.25528411],  #random
            [ 0.49313049,  0.94909878]]) #random
     """
+    if len(dn) == 1 and isinstance(dn[0], (tuple, list)):
+        raise TypeError('this function cannot take tuple as argument')
```

Comment:
I have tried numpy, and I got

```
In [1]: import numpy as np

In [2]: np.random.rand((10, 20))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-0fb309c754b2> in <module>()
----> 1 np.random.rand((10, 20))

mtrand.pyx in mtrand.RandomState.rand()

mtrand.pyx in mtrand.RandomState.random_sample()

mtrand.pyx in mtrand.cont0_array()

TypeError: 'tuple' object cannot be interpreted as an integer
```

What's the reason that we do not follow the exact error message?

Reply:
OK, i will change it to be consistent with numpy.